.mailmap

```diff
@@ -55,6 +55,7 @@ Garrison Taylor <garrisontaylor@gmail.com> gbear605 <garrisontaylor@gmail.com>
 Gulshan Kumar <gulshankumar1210iiit@gmail.com> Gulshan Mittal <gulshankumar1210iiit@gmail.com>
 Gulshan Kumar <gulshankumar1210iiit@gmail.com> gulshan-mittal <gulshankumar1210iiit@gmail.com>
 Guntbert Reiter <guntbert@gmx.at> G. Reiter <guntbert@gmx.at>
+Hannah Collier <76849369+hannahc243@users.noreply.github.com> Hannah Collier <hannah.collier@fhnw.ch>
```

Comment:
This feels like the reverse of what we want.  Doesn't the proper email address (hannah.collier@fhnw.ch) come first?

Reply:
Oh yeah 🤦 , I always forget... thanks for catching